packages/gatsby-source-vtex/src/gatsby-node.ts

```diff
@@ -186,12 +197,27 @@ export const sourceNodes: GatsbyNode['sourceNodes'] = async (
     .map(normalizePath)
     .filter((path) => !ignorePaths.includes(path))
 
-  const pageTypes = await pMap(
-    staticPaths,
-    (path: string) =>
-      fetchVTEX<PageType>(api.catalog.portal.pageType(path), options),
-    { concurrency }
-  )
+  const fetchPageTypes = async (path: string): Promise<PageType> => {
+    const isProduct = /\/(.)+\/p$/g
```

Comment:
This REGEX don't match with this kind of expression `/produto/p?whatever`

Reply:
Not really, we don't really have these kind of paths in here. We only have `/:slug/p`

Reply:
Ok ok, so sounds good.